worker.go

```diff
@@ -188,10 +188,16 @@ func (w *worker) processJob(job *Job) {
 		w.deleteUniqueJob(job)
 	}
 	if jt, ok := w.jobTypes[job.Name]; ok {
+		if jt.StartingDeadline > 0 && job.ScheduledAt > 0 && job.ScheduledAt < jt.StartingDeadline {
+			logInfo("Outdated! StartingDeadline: %d, ScheduledAt: %d, Job: %s\n", jt.StartingDeadline, job.ScheduledAt, job)
```

Comment:
We don't want to be logging info messages in the middle of a job processing loop - please remove this (it's not bad thing to have for running the tests).

Reply:
ok